platform/mellanox/sonic-bfb-installer

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+command_name="sonic-bfb-installer"
+usage(){
+    echo "Syntax: $command_name -b|--bfb <BFB_Image_Path> --rshim|-r <rshim1,..rshimN> --verbose|-v --config|-c <Options> --help|h"
+    echo "Arguments:"
+    echo "-b			Provide custom path for bfb image"
+    echo "-r			Install only on DPUs connected to rshim interfaces provided, mention all if installation is requried on all connected DPUs"
+    echo "-v			Verbose installation result output"
+    echo "-c			Config file"
+    echo "-h		        Help"
+}
+
+bfb_install_call(){
+    #Example:sudo bfb-install -b <full path to image> -r rshim<id>
+    local result_file=$(mktemp "/tmp/result_file.XXXXX")
+    local cmd="sudo ./bfb-install -b $2 -r $1 $appendix"
```

Comment:
I think it's better to make sonic-bfb-installer to be called by sudo and make sure privilege passes down to bfb-install script

Reply:
Done